test/functional/feature_init.py

```diff
@@ -57,19 +57,23 @@ def check_clean_start():
             assert_equal(200, node.getblockcount())
 
         lines_to_terminate_after = [
+            'Validating signatures for all blocks',
             'scheduler thread start',
+            'Starting HTTP server',
             'Loading P2P addresses',
             'Loading banlist',
             'Loading block index',
             'Switching active chainstate',
+            'Checking all blk files are present',
             'Loaded best chain:',
             'init message: Verifying blocks',
+            'init message: Starting network threads',
+            'net thread start',
+            'addcon thread start',
             'loadblk thread start',
             # TODO: reenable - see above TODO
             # 'txindex thread start',
-            'net thread start',
-            'addcon thread start',
-            'msghand thread start',
+            'msghand thread start'
```

Comment:
nit: Any reason to remove the trailing comma? This makes this diff (and potential future ones) larger than necessary.

Reply:
@MarcoFalke you are right. Having the comma would have been a better choice, as you are pointing. I see PR is merged already but will keep this in mind for next times. Thank you!